pkgs/applications/editors/abricotine/default.nix

```
@@ -0,0 +1,78 @@
+{ stdenv, lib, fetchurl, copyDesktopItems, makeDesktopItem, makeWrapper
+, writeText , electron }:
+
+let
+  # This script is mandatory to make this application work
+  # without user intervention. The reasoning is specified below:
+  # https://github.com/NixOS/nixpkgs/issues/148152#issuecomment-983773905
+  exec_script = writeText "exec_script" ''
+    #!/usr/bin/env bash
+
+    set -euo pipefail
+
+    CONFIG_PATH="$HOME/.config/Abricotine"
+
+    if [ ! -d "$CONFIG_PATH" ]; then
+        SOURCE_SUBPATH="/opt/abricotine/resources/app.asar.unpacked/default"
+        mkdir -p "$CONFIG_PATH/app"
+        cp -R "$1$SOURCE_SUBPATH/dict" "$CONFIG_PATH/app/dict"
+        cp -R "$1$SOURCE_SUBPATH/lang" "$CONFIG_PATH/app/lang"
+        cp -R "$1$SOURCE_SUBPATH/templates" "$CONFIG_PATH/app/templates"
+        cp -R "$1$SOURCE_SUBPATH/themes" "$CONFIG_PATH/app/themes"
+        chmod -R u+w "$CONFIG_PATH/app/"
+    fi
+  '';
+
+in stdenv.mkDerivation rec {
+  pname = "abricotine";
+  version = "1.0.0";
+
+  src = fetchurl {
+    url = "https://github.com/brrd/abricotine/releases/download/${version}/${pname}-${version}-linux-x64.tar.gz";
```

The usage of `${pname}` makes it a bit harder to read IMO, with seemingly no benefits.
```suggestion
    url = "https://github.com/brrd/abricotine/releases/download/${version}/abricotine-${version}-linux-x64.tar.gz";
```

---

Agreed. I've changed it.